packages/test/jest.test.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-env nodes, jest */
+
+const hrp = require("http-request-plus").default;
+const jrp = require("json-rpc-protocol");
+
+async function call(method, params) {
+  return jrp.parse.result(
+    await hrp
+      .post("http://localhost:4000/api/", {
+        body: jrp.format.request(0, method, params),
+      })
+      .readAll("utf-8")
+  );
+}
+
+const invertPromise = p =>
+  p.then(
+    value => {
+      throw value;
+    },
+    reason => reason
+  );
+
+test("create entry", async () => {
+  expect(
+    await call("createEntry", { name: "name1", content: "content1" })
+  ).toBe(1);
+  expect(await call("createEntry", { content: "content2" })).toBe(2);
+  expect(await call("createEntry", { name: "name3" })).toBe(3);
+
+  const response = await call("listEntries");
+
+  expect(response).toEqual([
+    {
+      id: 1,
+      name: "name1",
+      content: "content1",
+      created: expect.any(Number),
+      updated: expect.any(Number),
+    },
+    {
+      id: 2,
+      content: "content2",
+      created: expect.any(Number),
+      updated: expect.any(Number),
+    },
+    {
+      id: 3,
+      name: "name3",
+      created: expect.any(Number),
+      updated: expect.any(Number),
+    },
+  ]);
+});
+
+test("update entry", async () => {
+  await call("updateEntry", {
+    id: 1,
+    name: "name1_modified",
+    content: "content1_modified",
+  });
+  await call("updateEntry", { id: 2, name: "only_name2_modified" });
+  await call("updateEntry", { id: 3, content: "only_content3_modified" });
+
+  const response = await call("listEntries");
+
+  expect(response).toEqual([
+    {
+      id: 1,
+      name: "name1_modified",
+      content: "content1_modified",
+      created: expect.any(Number),
+      updated: expect.any(Number),
```

Comment:
We should test:
- `created` has not been changed
- the new value of `updated` is coherent